browser/brave_stats/brave_stats_updater_params.cc

```diff
@@ -105,8 +105,10 @@ std::string BraveStatsUpdaterParams::GetProcessArchParam() const {
 }
 
 std::string BraveStatsUpdaterParams::GetWalletEnabledParam() const {
-  uint8_t usage_bitset = UsageBitfieldFromTimestamp(
-      wallet_last_unlocked_, last_reported_wallet_unlock_);
+  uint8_t usage_bitset = 0;
+  if (is_last_wallet_unlock_v2_)
+    usage_bitset = UsageBitfieldFromTimestamp(wallet_last_unlocked_,
+                                              last_reported_wallet_unlock_);
```

Comment:
Curly braces [around the conditional body](https://google.github.io/styleguide/cppguide.html#Conditionals), please. Direct bodies like this are harder to read an [error prone](https://www.imperialviolet.org/2014/02/22/applebug.html).